homeassistant/components/venstar/sensor.py

```diff
@@ -0,0 +1,138 @@
+"""Representation of Venstar sensors."""
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    STATE_CLASS_MEASUREMENT,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, TEMP_CELSIUS, TEMP_FAHRENHEIT
+
+from . import VenstarDataUpdateCoordinator, VenstarEntity
+from .const import DOMAIN
+
+
+@dataclass
+class VenstarSensorEntityDescription(SensorEntityDescription):
+    """Base description of a Sensor entity."""
+
+
+HUMIDITY_SENSOR_DESCRIPTION = VenstarSensorEntityDescription(
+    key="humidity",
+    device_class=DEVICE_CLASS_HUMIDITY,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=PERCENTAGE,
+)
+TEMPERATURE_SENSOR_DESCRIPTION = VenstarSensorEntityDescription(
+    key="temperature",
+    device_class=DEVICE_CLASS_TEMPERATURE,
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
+    """Set up Vensar device binary_sensors based on a config entry."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+    hum_sensors = []
+    temp_sensors = []
```

Comment:
```suggestion
    entities = []
```

Reply:
error: Argument 1 to "append" of "list" has incompatible type "VenstarTemperatureSensor"; expected "VenstarHumiditySensor"  [arg-type]

from mypy...